<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNAExportDialog</class>
 <widget class="QDialog" name="GNAExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to GNA GeoPackage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <!-- Project Section -->
    <widget class="QGroupBox" name="groupBox_project">
     <property name="title">
      <string>Project</string>
     </property>
     <layout class="QFormLayout" name="formLayout_project">
      <item row="0" column="0">
       <widget class="QLabel" name="label_project_code">
        <property name="text">
         <string>Project Code:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_project_code">
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_project_title">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_project_title"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_responsible">
        <property name="text">
         <string>Responsible:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_responsible"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- Project Area Section -->
    <widget class="QGroupBox" name="groupBox_area">
     <property name="title">
      <string>Project Area (MOPR)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_area">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_polygon">
        <item>
         <widget class="QLabel" name="label_polygon_source">
          <property name="text">
           <string>Polygon Source:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_polygon_layer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_feature">
        <item>
         <widget class="QLabel" name="label_feature">
          <property name="text">
           <string>Feature:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_polygon_feature">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_select_from_map">
        <property name="text">
         <string>Select from Map...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- Layers Section -->
    <widget class="QGroupBox" name="groupBox_layers">
     <property name="title">
      <string>Layers to Export</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_layers">
      <item>
       <widget class="QCheckBox" name="checkBox_mosi">
        <property name="text">
         <string>MOSI - Archaeological Sites</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_vrp">
        <property name="text">
         <string>VRP - Archaeological Potential</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_vrd">
        <property name="text">
         <string>VRD - Archaeological Risk</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- Heatmap Options Section -->
    <widget class="QGroupBox" name="groupBox_heatmap">
     <property name="title">
      <string>Heatmap Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_heatmap">
      <item row="0" column="0">
       <widget class="QLabel" name="label_method">
        <property name="text">
         <string>Interpolation Method:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_method">
        <item>
         <property name="text">
          <string>KDE (Kernel Density)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IDW (Inverse Distance)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Grid</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_cell_size">
        <property name="text">
         <string>Cell Size (m):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_cell_size">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- Output Section -->
    <widget class="QGroupBox" name="groupBox_output">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_output">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_output">
        <item>
         <widget class="QLabel" name="label_output_path">
          <property name="text">
           <string>Output Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_output_path">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_browse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_load_layers">
        <property name="text">
         <string>Load layers in QGIS after export</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- Progress Section -->
    <widget class="QGroupBox" name="groupBox_progress">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_progress">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_status">
        <property name="text">
         <string>Ready</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <!-- Button Box -->
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GNAExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GNAExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
